.github/workflows/CreateRelease.yml

```diff
@@ -89,40 +103,46 @@ jobs:
           Write-Host "msiexec exited with code $LASTEXITCCODE"
           if ($LASTEXITCODE -ne 0) { cat log.txt; exit 1 }
     - name: Create pre-release
-      if:  (!contains(github.ref, 'refs/heads/release/')) &&  matrix.config == 'debug' 
+      if: ${{ github.ref=='refs/heads/main' }}
       run:  |
             echo "PWD: $PWD"
             ls -alR
             gh release delete dev-latest -y --cleanup-tag  || true
             gh release create dev-latest -t "Latest Development Build From Dev Branch" --latest=false -p \
-                ${{ env.PLATFORM }}/${{ matrix.config }}/HelloWorld.wasm \
-                ${{ env.PLATFORM }}/${{ matrix.config }}/RunWasm.wasm \
-                ${{ env.PLATFORM }}/${{ matrix.config }}/HelloWorld.aot \
-                ${{ env.PLATFORM }}/${{ matrix.config }}/wasm_runtime.* \
-                ${{ env.PLATFORM }}/${{ matrix.config }}/rust_wasm_samples.aot \
-                ${{ env.PLATFORM }}/${{ matrix.config }}/rust_wasm_samples.wasm \
+                ${{ env.PLATFORM }}/${{ env.CONFIG }}/HelloWorld.wasm \
```

Comment:
Looks good, but I was thinking maybe we don't need to publish all these things. Perhaps something we can consider in a follow up issue/pr

Reply:
I guess we don't really need to do a GH release then? Just the crates.io publishing?

Reply:
I think it's useful to have the release so its easy to track changesets. 

The `aot` files specifically won't be all that useful since they are compiled to a specific platform/wasmtime version.

Reply:
That's a very good point. I'll create the release but without uploading any extra artifacts (I think GH adds a tar with the source by default)